_rules/audio-transcript-2eb176.md

```diff
@@ -81,7 +88,7 @@ This `audio` element has native player controls and an external transcript.
 
 #### Passed Example 3
 
-This `audio` element has an `autoplay` attribute and an external transcript.
+This `audio` element has an [autoplay][] attribute and an external transcript.
```

Comment:
Since autoplay here is referring to the HTML attribute I don't think it needs to link to the definition.